kong/plugins/hmac-auth/daos.lua

```diff
@@ -1,3 +1,9 @@
+local utils = require "kong.tools.utils"
+
+local function random_secret(t)
+  return utils.random_string()
+end
```

Comment:
why wrapping a single function call into another function?

if it is to drop the argument, then maybe a small comment would make that clear